### PR TITLE
fix: fixed schedule only query issue

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
- Modification of only data requested when adjusting the schedule.

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-main/CollectorPage.vue [hunk 1]
```diff
@@ -152,6 +153,7 @@ const refreshCollectorList = async () => {
     state.initLoading = false;
     state.hasCollectorList = true;
     collectorApiQueryHelper.setFilters(collectorPageStore.allFilters);
+    console.log(collectorApiQueryHelper.data);
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-main/CollectorPage.vue:156]: suggestion: Please remove console.log

**Review by WANZARGEN — APPROVED** [review]
LGTM

File: apps/web/src/services/asset-inventory/collector/collector-main/CollectorPage.vue (hunk 1 above)
Comment (skdud4659) [reply apps/web/src/services/asset-inventory/collector/collector-main/CollectorPage.vue:156]: oops!

**Review by yuda110 — APPROVED** [review]
LGTM!!

**Pull request merged**